estate/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
+    'name': "Real estate",
```

Comment:
```suggestion
    'name': 'Real estate',
```
Prefer using double quotes ("") for long strings, multiline or user inputs/displays.